src/platform/symbols.map

```diff
@@ -10,14 +10,15 @@ MIR_PLATFORM_2.17 {
     mir::graphics::DMABufEGLProvider::DMABufEGLProvider*;
     mir::graphics::DMABufEGLProvider::as_texture*;
     mir::graphics::DRMFormat::DRMFormat*;
-    mir::graphics::DRMFormat::alpha_equivalent*;
+    mir::graphics::DRMFormat::Info::alpha_equivalent*;
     mir::graphics::DRMFormat::as_mir_format*;
-    mir::graphics::DRMFormat::components*;
+    mir::graphics::DRMFormat::Info::components*;
     mir::graphics::DRMFormat::from_mir_format*;
-    mir::graphics::DRMFormat::has_alpha*;
+    mir::graphics::DRMFormat::Info::has_alpha*;
     mir::graphics::DRMFormat::name*;
-    mir::graphics::DRMFormat::opaque_equivalent*;
+    mir::graphics::DRMFormat::Info::opaque_equivalent*;
```

Comment:
Is this a breaking change for the symbols since we're deleting some symbols? Would we need to bump the major version in this case?

Reply:
Yes, thanks for the catch; I have now done so!